### PR TITLE
D42-11958

### PR DESCRIPTION
**Pull request by osanchez42**
Cisco Support (Not tested)

### CODE REVIEW

File: Files/warranty.cfg.example [hunk 1]
```diff
@@ -6,13 +6,20 @@ url =
 
 [discover]
 # enable/disable discoveries
-dell = True
-hp = True
-ibm = True
-lenovo = True
-meraki = True
+cisco = True
+dell = False
```
Comment (cscaglioned42) [inline Files/warranty.cfg.example:10]: Shouldn't all the vendors be set to True?

File: README.md [hunk 2]
```diff
@@ -21,9 +22,6 @@ In order for this script to check warranty status of the device, the device must
 - IBM script points to warranty info not related to the SKU, serial given
 - If a Meraki product has a licence with a renewal required state, the expiration date will be set to the current date
 
-## Change Log
```
Comment (cscaglioned42) [inline README.md:24]: Why was this removed?

File: Files/warranty_cisco.py [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+
+            self.get_access_token(self.client_id, self.client_secret)
+
+        if self.access_token is None:
+            if self.debug > 1:
+                print 'unable to acquire access_token'
+            return None
+
+        # get the device information using the requested access token
+        # maximum serials per request is 70 devices with a maximum length of 40 separated by commas
```
Comment (cscaglioned42) [inline Files/warranty_cisco.py:108]: Comment says we can only make a request with 70 devices.  Do we enforce this?

Comment (osanchez42) [reply Files/warranty_cisco.py:108]: ` api_path = '/api/1.0/devices/all/'
        cols = '?include_cols=serial_no,device_id,manufacturer&limit=50&offset=' + str(offset) + '&hardware=' + models
        path = api_path + cols
        response = self.get_data(path)`

Hey Chris, the standalone script if I am correct is getting devices with an offset of 50 for all vendors

File: README.md (hunk 2 above)
Comment (osanchez42) [reply README.md:24]: I will add it back for future changes but at the time it was not being used

File: Files/warranty_cisco.py (hunk 3 above)
Comment (cscaglioned42) [reply Files/warranty_cisco.py:108]: Yes, it is 50.

File: Files/warranty.cfg.example (hunk 1 above)
Comment (osanchez42) [reply Files/warranty.cfg.example:10]: set back to true
